osu.Game/Overlays/Dialog/PopupDialog.cs

```diff
@@ -34,26 +37,28 @@ public class PopupDialog : OsuFocusedOverlayContainer
         private readonly SpriteText header;
         private readonly TextFlowContainer body;
 
+        private bool actionInvoked;
```

Comment:
Why do we need this variable?

Reply:
It avoids actions potentially being triggered twice on hiding (since there is now added invocation logic in the `PopOut` method).

Reply:
Added an inline comment pertaining to this.